hack/shellcheck-test.sh

```diff
@@ -63,5 +65,5 @@ done
 
 echo "${failed} scripts with errors were found"
 
-exit ${failed}
+exit "${failed}"
```

Comment:
Separate commit, can be the same as the whitespace fix.

Reply:
done